docs/tutorials/java-troubleshooting.rst

```diff
@@ -0,0 +1,33 @@
+Java Troubleshooting
+######################################################
+
+In this tutorial, you will use Robusta to troubleshoot a java application running on Kubernetes. Specifically,
+you will be remotely running functions from a custom JDK on any Java application.
+
+Doing it
+---------------------------------------
+
+Let's run the ``java_process_inspector`` action builtin to Robusta.
+
+| You need a Java application to debug. Look at ``test_pod_config.yaml`` in Robusta's ``java_toolkit`` repository for a demo Java application.
```

Comment:
Instead of telling people where to look can you make this:

Deploy a test Java application: ``kubectl apply -f gist.github....`